zio-http/src/main/scala/zio/http/FormField.scala

```diff
@@ -155,15 +155,15 @@ object FormField {
           Chunk.empty[FormAST.Content],
         ),
       ) {
-        case (accum, header: FormAST.Header) if header.name == "Content-Disposition"       =>
+        case (accum, header: FormAST.Header) if header.name.toLowerCase == "content-disposition"       =>
```

Comment:
I would recommend using the `equalsIgnoreCase` method on String instead. Same for the other cases
```suggestion
        case (accum, header: FormAST.Header) if header.name.equalsIgnoreCase("Content-Disposition")        =>
```